tests/functional/incremental-fetching-date-with-state/expected/data/out/tables/in.c-main.auto-increment-timestamp.csv.manifest

```diff
@@ -62,7 +58,7 @@
             },
             {
                 "key": "KBC.ordinalPosition",
-                "value": 0
+                "value": 1
```

Comment:
`ordinalPosition` je cislovana od `1`, tak to ma aj `db-ex-mysql`, takze by to malo byt spravne.